plover/oslayer/osxkeyboardcontrol.py

```diff
@@ -344,7 +365,10 @@ def send_key_combination(self, combo_string):
         # that, if executed in order, would emulate the key combination
         # represented by the argument.
         def _keystring_to_sequence(keystring):
-            if keystring in KEYNAME_TO_KEYCODE:
+            if keystring in NX_KEYS:
+                self._send_media_key(NX_KEYS[keystring])
+                seq = []
```

Comment:
Could not this code result in sending key events in the wrong order? E.g. when using something like `{#RegularKey MediaKey}`?